chatmeleon-server/libs/prismadb.ts

```diff
@@ -0,0 +1,17 @@
+// Import the PrismaClient from the "@prisma/client" package
+import { PrismaClient } from "@prisma/client";
+import 'dotenv';
+
+// Declare a global variable for the PrismaClient
+declare global {
+    var prisma: PrismaClient | undefined;
+}
+
+// Initialize a PrismaClient instance, or use an existing one if available globally
+const client = globalThis.prisma || new PrismaClient();
+
+// If not in production mode, assign the PrismaClient instance to the global variable
+if (process.env.NODE_ENV !== 'production') globalThis.prisma = client;
+
```

Comment:
Remove globalThis definitions if not used

Reply:
Are you sure? Ehe

Reply:
Yeah. From what I’ve seen. All prisma references are from the exported prisma client from this file. The globalThis reference is unused and confusing

Reply:
oke ehe